ansible/playbooks/koala/activestorage-migration.yml

```diff
@@ -0,0 +1,95 @@
+---
+- hosts: "all"
+  remote_user: "ansible"
+  force_handlers: true
+  become: true
+  become_user: "root"
+
+  vars_prompt:
+    - name: "confirmation"
+      default: "n"
+      prompt: "Are you sure? 'Yes' to continue."
+      private: false
+
+  tasks:
+    - assert:
+        that:
+          - "confirmation == 'Yes'"
+
+    - name: "install new dependency"
```

Comment:
Surely this won't be a 'new' dependency forever

Reply:
It will be for the lifetime of this playbook, which is very hopefully one execution. This playbook is needed only once for a 'special' migration, which would break if we did it using the normal way (another command needs to be run), and I didn't want to alter the main playbook for this.